crankshaft-config/src/backend/defaults.rs

```diff
@@ -37,11 +48,21 @@ impl Defaults {
         self.cpu
     }
 
+    /// Gets the CPU limit.
+    pub fn cpu_limit(&self) -> Option<f64> {
+        self.cpu_limit
+    }
+
     /// Gets the amount of RAM (in GB).
```

Comment:
Perhaps I also missed these going from GB -> GiB?

Reply:
They weren't used in any backends yet, so I don't think we had to adjust the conversion, but I'll correct the comment to make it consistent.

Reply:
Yes, that was my thought too. Sounds good. I'd hit on all of the comments for getters that are related to this.